src/init.c

```diff
@@ -298,6 +298,9 @@ shmem_internal_init(int tl_requested, int *tl_provided)
             printf("SMA_INFO                %s\n",
                    (NULL != shmem_util_getenv_str("INFO")) ? "Set" : "Not set");
             printf("\tIf set, print this help message at startup\n");
+            printf("SMA_OFI_ATOMIC_CHECKS_WARN  %s\n",
```

Comment:
I don't think we want this at the top level...  The way it's written right now, the OFI transport info would also be printed out in the Portals build, even though the env var has no effect.  I think it would be better to have a transport layer function that gets called during initialization to print out an info message.